rosdep/base.yaml

```diff
@@ -3160,8 +3160,13 @@ linux-headers-generic:
     jessie: [linux-headers-3.16.0-4-all]
     stretch: [linux-headers-4.9.0-2-all]
     wheezy: [linux-headers-3.2.0-4-all]
-  fedora: [kernel-headers]
+  fedora: [kernel-headers, kernel-devel]
   ubuntu: [linux-headers-generic]
+linux-kernel-headers:
+  arch: [linux-api-headers]
+  debian: [linux-kernel-headers]
```

Comment:
Trying this out on jessie: 
```
root@57d6a0a989c1:/# apt-cache search linux-kernel-headers
linux-libc-dev - Linux support headers for userspace development
root@57d6a0a989c1:/# apt install linux-kerner-headers
Reading package lists... Done
Building dependency tree... Done
E: Unable to locate package linux-kerner-headers
```
So I think we need to split jessie and stretch and use `linux-libc-dev` for jessie